### DISCUSSION TITLE
Help w/ spread operator in RunJS

### DISCUSSION THREAD

**Discussion by rsarvis** [lukehaas/RunJS#661] [General]
Sorry if this is not the proper place to ask this but...

I'm using RunJS to play with a line of code, but I'm getting an error that comes from trying to use the spread operator, so not sure what I'm doing wrong.

Shouldn't this work?
`[...Array(10).keys()]`

I just get `SyntaxError: Unexpected token (...:1)` so it's not accepting the Spread Operator. Is it a matter of the version of the language I'm using, and if so, how do I change that?

**Comment by lukehaas** [comment]
@rsarvis I'm not seeing any issues when running this:

![Screenshot 2024-11-21 at 09 44 02](https://github.com/user-attachments/assets/79cf535b-d763-4a45-bb90-3e120ac9d5a3)

What version of RunJS are you using?

**Comment by lukehaas** [comment]
@rsarvis any update on this?
